test_util/test_aws_cf.py

```diff
@@ -55,6 +60,13 @@ def check_environment():
     # Defaults
     options.ci_flags = os.getenv('CI_FLAGS', '')
     options.aws_region = os.getenv('DEFAULT_AWS_REGION', 'eu-central-1')
+    options.ensure_dcos_cleanup = os.getenv('ENSURE_DCOS_CLEANUP', 'true') == 'true'
+    options.advanced = os.getenv('DCOS_ADVANCED_TEMPLATE', 'false') == 'true'
```

Comment:
Oh, wait, nope, cant do that if we want to be able to attach to an already running advanced stack by stack name, so I guess this stays as is?